client/post-editor/editor-revisions-list/index.jsx

```diff
@@ -49,13 +50,18 @@ class EditorRevisionsList extends PureComponent {
 	}
 
 	render() {
+		const usersId = uniq( map( this.props.revisions, r => get( r, 'author.ID', r.author ) ) );
```

Comment:
Not sure about the third parameter of the `get ` call here. Should we drop it or am I missing something?